src/main/java/org/broadinstitute/hellbender/tools/copynumber/caller/SimpleCopyRatioCaller.java

```diff
@@ -0,0 +1,135 @@
+package org.broadinstitute.hellbender.tools.copynumber.caller;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.tools.copynumber.formats.collections.CalledCopyRatioSegmentCollection;
+import org.broadinstitute.hellbender.tools.copynumber.formats.collections.CopyRatioSegmentCollection;
+import org.broadinstitute.hellbender.tools.copynumber.formats.records.CalledCopyRatioSegment;
+import org.broadinstitute.hellbender.tools.copynumber.formats.records.CopyRatioSegment;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.param.ParamUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ * This caller is loosely based on the legacy ReCapSeg caller that was originally implemented in ReCapSeg v1.4.5.0,
+ * but introduces major changes.  The method is as follows:
+ * 1) use the non-log2 mean copy ratio to determine copy-neutral segments,
```

Comment:
If this is going to be published as javadoc it needs to use html list elements, e.g. `<ol><li>...<li>...</ol>`. Otherwise this will all be formatted as a single paragraph, with lines wrapped together.

Reply:
Thanks @pshapiro4broad, doc updates are going to be handled in a separate PR.  Also, I think we are typically less careful about formatting javadoc that is not for CLI classes, since these will be geared more towards developers.